rpc2/example/client.go

```diff
@@ -14,32 +14,33 @@ type GenericClient interface {
 //---------------------------------------------------------------------
 
 type ArithClient struct {
-	cli GenericClient
+	GenericClient
```

Comment:
Just a stylistic preference: An `ArithClient` is a `GenericClient`, so it should adhere to the API.